### DISCUSSION TITLE
Unable to REVerify as a Student

### DISCUSSION THREAD

**Discussion by Nailington** [community/community#53827] [GitHub Education]
### Hi everyone,

I am a Junior currently in the FY22-23 school year. I attend a public high school that is listed in the drop down list. I am currently (but lose access to) marked as student with GitHub. I am unable, no matter how hard I try, able to get verified. I have used many different types of verification, used live photos, used uploaded images, used OBS Virtual Camera for a clear SS, used my primary email that is attached to my account, used a different secondary email. And before someone asks if I am using my School Email Address, the answer is no, because I am unable to receive emails from external sources. I have tried nearly EVERYTHING possible. I have attempted 14 times since the 4th of April.

 In this image
![last 4 attempts](https://user-images.githubusercontent.com/111466491/234703673-d95637b8-1c92-4435-b1ed-a06fccc2be04.png), I have the last 4 attempts. All of the attempts say either of the two, though recently it has been leaning towards the "Common issues" link. 

My name is my account, along with my school (despite not wanting this info public). I have a pfp (while not my face, I am not comfortable sharing my face online just so I can be a student. It should be able to see my face in the ID photo verification) on my account. I am already verified in the past (and technically as of this post still verified until the 3rd of May). WHY am I unable to be verified.

I have had errors while doing the application like this: ![example](https://user-images.githubusercontent.com/111466491/234705319-4e2e3cfb-0818-42c5-b743-cb5f2b0eac61.png)

and this: ![image](https://user-images.githubusercontent.com/111466491/234706069-ecc91d6a-8feb-4649-abdf-8ee0fd8338eb.png)

and this: ![image](https://user-images.githubusercontent.com/111466491/234706148-913b8198-71b9-4d16-beb0-dd8b1e6b8557.png)

and this: ![image](https://user-images.githubusercontent.com/111466491/234706247-34d62ea3-ea67-44ff-a810-12e62fa1c0a3.png)

I am legit unable to verify, no matter how hard I try. What more do I need to do? I would request that someone give me a manual review, but from everything I have seen, there aren't any to be had. I just want to continue to use these services while I am a student as I am entitled. 

Overall, sorry for the rant like post, but in my opinion, it is frankly unacceptable that the system for verification is flawed. It would be one thing if I was a one off situation, but not only does everyone I know IRL, or even people I know online many states away, have the SAME EXACT issues, along with the many posts I have seen this year on various parts of the web. I just want to be verified, that's it.

**Comment by samus-aran** [comment]
Hi @Nailington 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. 

We are extremely proud that the GitHub Global Campus has provided _millions_ of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 
However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. **Student ID**) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get along 👍

**Reply by Cattn** [reply]
I can second having the same issue, I was able to get in (instantly albiet) when I signed up about 2 years ago, with a OBS virtual camera image of my school portal (it has my school info, name, photo, classes, all of that) but while trying to do it with the same exact screenshot (down to all the dimensions, as well as content) IT also includes Enrollment year, I was denied. I've held back from trying many times so as to not potentially get limited in some way, and would rather contact here first, if there's anything that can be done about it outside of the system as it doesn't seem to work for me

**Reply by samus-aran** [reply]
Hi @Cattn 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. Remember to include your rejection reasons 👍

**Reply by Nailington** [reply]
I have tried this, I have used a schedule, SIS, School ID, Portal SS, picture of an SS, and more. None of these attempts have worked.
In fact, I have been errored out by NOT using an ID. For example:
<img width="279" alt="image" src="https://user-images.githubusercontent.com/111466491/234712108-cf1b5007-6d1c-4af5-ab75-f6037cfa069d.png">

I understand that not having a school email may impede progress, but I have gone up and down the type of verification list, nothing is working. I just want to be verified, that's it. I have followed all of the FAQs and guidance. What more do I need to verify myself as a student. I legit cannot do anything more.

**Reply by samus-aran** [reply]
We appreciate this but it depends on the school. If the system has detected the school has evidence of distributing particular types of evidence we do ask students to provide these.

Can I ask, what is the last rejection reason you received and we can begin from there.

**Reply by Nailington** [reply]
Less than two hours ago: 
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/111466491/234715487-ca049907-9032-4338-a654-fac22503f509.png"> This was with a School ID, and my primary email.

There is no helpful description in this reason for being denied. I have followed this GH Docs page, and as stated earlier, I have tried all of the different document types. Also as stated earlier, I am unable to provide my school email address.

**Reply by Nailington** [reply]
@samus-aran Any updates by chance? I lose my status on the 3rd, and rather not lapse in my eligibility.

**Reply by jgonzalezalbisu** [reply]
I'm having the same problem https://github.com/orgs/community/discussions/53695

**Reply by Nailington** [reply]
> @samus-aran Any updates by chance? I lose my status on the 3rd, and rather not lapse in my eligibility.

I had my Student Status expire yesterday. I tried again just now while on campus, on a school device, using my ID, and it has failed with no explanation again...
<img width="225" alt="image" src="https://user-images.githubusercontent.com/111466491/236220290-58fc7ac3-6d19-4bc1-bb4d-17740e99068e.png">
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/111466491/236220749-76e13998-bc51-4823-bc51-b28eec020fd7.png">
I just want to be able to have the benefits I am entitled to as a student.

**Reply by jgonzalezalbisu** [reply]
It seems that the benefit is going to cease to be

**Reply by Nailington** [reply]
@samus-aran, I am just checking in again because I am still unable to verify while on campus, with various forms of identification. I have seen other people verify instantaneously using the same exact tools I have at my school. I don't want to seem as a pest, but frankly it should not require this much work, frustration, and aggravation to verify for something when it is very clearly obvious that I am a student. Here is my attempt from within 30 mins: 
<img width="1025" alt="image" src="https://github.com/community/community/assets/111466491/aa1eb8fd-90a3-4c3f-aade-825531162db5">


It is ridiculous that there is no support. Not only is there no way to contact support about educational subjects (including verification), but you must rely on a discussion like so, which means no matter what, we are forced to not actually get help. And then these discussions are left in the dust, and forgotten. This shouldn't be an issue, and is very annoying and unnecessary for something so simple.

**Reply by jgonzalezalbisu** [reply]
Still no answer?

**Reply by Cattn** [reply]
nope
